app/controllers/admin/events_controller.rb

```diff
@@ -17,7 +17,7 @@ def set_timezone_for_this_request(&block)
 
     def index
       @events = @program.events
-      @tracks = @program.tracks
+      @tracks = @program.tracks.confirmed.cfp_active
```

Comment:
I am not sure we need the tracks to have cfp_active, so that we are able to move events from one track to another during evaluation period.
Let's look into that later on.